test/e2e/specs/importer/importer__import-focused.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * @group calypso-pr
+ */
+
+import { DataHelper, StartImportFlow, TestAccount, SecretsManager } from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+import type { TestAccountName } from '@automattic/calypso-e2e';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Stepper: setup/import-focused' ), () => {
+	const credentials = {
+		simpleSiteFreePlanUser: SecretsManager.secrets.testAccounts.simpleSiteFreePlanUser,
+		jetpackRemoteSiteUser: SecretsManager.secrets.testAccounts.jetpackRemoteSiteUser,
+	};
+
+	let page: Page;
+	let startImportFlow: StartImportFlow;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		startImportFlow = new StartImportFlow( page );
+	} );
+
+	const loginAsUser = ( user: TestAccountName ) => {
+		it( `Login as ${ user }`, async () => {
+			const testAccount = new TestAccount( user );
+			await testAccount.authenticate( page );
+		} );
+	};
+
+	const navigateToImportFocusedFlow = ( step: string, siteSlug?: string, from?: string ) => {
+		it( `Navigate to page setup/import-focused?from=${ from }`, async () => {
```

Comment:
Maybe we can add the step here as well, so by looking into the test it's more clear?

Reply:
True! Done.